software/validation/stellar_density_comparisons.py

```diff
@@ -0,0 +1,132 @@
+"""
+This script reads in the HEALPIX maps generated by
+
+stellar_density_get_arrays.py
+stellar_density_control_arrays.py
+
+and creates plots of them.
+"""
+
+from __future__ import with_statement
+
+import matplotlib
+matplotlib.use('Agg')
+
+import numpy as np
+import healpy as hp
+import matplotlib.pyplot as plt
+
+import time
+
+import os
+
+if __name__ == "__main__":
+
+    t_start = time.time()
+
+    NSIDE = 64
```

Comment:
Why nside=64? Also, is it possible to make the maps at different resolutions, or is this set by the data? If it's set by the data, it would be good to instead calculate it from the data if possible.